tools/phpunit/bootstrap.php

```diff
@@ -58,13 +58,15 @@
 	$plugin_name = 'performance-lab';
 }
 
+define( 'TESTS_PLUGIN_DIR', TESTS_REPO_ROOT_DIR . "/plugins/$plugin_name" );
+
 /**
  * Load plugin bootstrap and any dependencies.
  *
  * @param string $plugin_name Plugin slug to load.
  */
 $load_plugin = static function ( string $plugin_name ) use ( &$load_plugin ): void {
-	$plugin_test_path = TESTS_PLUGIN_DIR . '/plugins/' . $plugin_name;
+	$plugin_test_path = TESTS_REPO_ROOT_DIR . '/plugins/' . $plugin_name;
```

Comment:
See above:
```suggestion
	$plugin_test_path = TESTS_PLUGIN_DIR;
```

Alternatively, we could get rid of this variable entirely, since it's only referencing the constant as-is.

Reply:
Actually, we can't do this. This function also needs to load plugin dependencies. I thought this at first myself.

Reply:
Which function do you mean?

Reply:
See this `$load_plugin` function here is used recursively. It loads any `Requires Plugins`.

Reply:
I'm not sure when the `$load_plugin` approach was added, just seeing this now.

It doesn't make sense to me to use this approach, especially given the complexity you're describing. There is a special global to control which plugins should be active. For example:

```php
$GLOBALS['wp_tests_options'] = array(
	'active_plugins' => array( basename( TESTS_PLUGIN_DIR ) . '/load.php' ),
);
```

Can we not use that approach? If the problem is that we're loading plugins from a custom subdirectory, I wonder whether we could maybe override `WP_PLUGIN_DIR` to be `TESTS_REPO_ROOT_DIR . '/plugins'`.

Overall, it certainly makes more sense to me to tie into WordPress's built-in plugin loading mechanism instead of trying to manually require files here (which BTW `plugins_loaded` is, strictly speaking, also too late for).

Reply:
The current approach seems to have been working well, but what you propose seems good. Maybe do it another PR?

Reply:
Makes sense, that would be a separate enhancement.